test/css/component_specific_selectors_test.rb

```diff
@@ -14,16 +14,34 @@ class ComponentSpecificSelectorsTest < Minitest::Test
   include Primer::RenderPreview
 
   IGNORED_SELECTORS = {
-    :global => [/^\d/, ":", /\[.*\]/],
-    Primer::Alpha::ActionList => [/^to/],
```

Comment:
I moved `/^to/` back to `:global` and also added `/^from/`. I think they are used for keyframe animations like:

```css
@keyframes checkmarkIn {
  from {
    clip-path: inset(16px 0 0 0);
  }

  to {
    clip-path: inset(0 0 0 0);
  }
}
```

and probably used for other components at some point.